app/models/differential_expression_parameters.rb

```diff
@@ -0,0 +1,67 @@
+# class to hold parameters specific to differential expression jobs in PAPI
+class DifferentialExpressionParameters
+  include ActiveModel::Model
+
+  # regular expression to validate GS url format
+  GS_URL_REGEXP = %r{\Ags://}.freeze
+
+  # annotation_name: name of annotation to use for DE
+  # annotation_type: type of annotation, must be 'group'
```

Comment:
That's a fair point - and you're right in that it is basically only a validation to avoid calling on a numeric annotation.  Mostly, I want to maintain parity between the interface here and in `scp-ingest-pipeline`, and have parameter names line up.  It seems somewhat strange to me to have an upstream validation on a parameter value when the rest are validated here.  I dunno - I could go either way, and this implementation has the benefit of being internally consistent and very compact.  But I agree it's weird to only allow a single value as an input.